### ISSUE TITLE
Cron: Error cleaning statlogs

### ISSUE THREAD

**Issue by WebSpider** [filesender/filesender#2010]
When running filesender3.0rc3 on php8.2.23 with a pgsql backend, we are getting a DB error cleaning the StatLog

Error from cron log is as follows, and appears to be a syntax error in the SQL query

```
[2024-09-20 10:04:07] [cron:info] [cron:info] Cleaning statlogs
[2024-09-20 10:04:07] [cron:info] [cron:info] Removing statlogs older than 60 days
[2024-09-20 10:04:07] [cron:error] [cron:error] [DBIUsageException exception uid:66ed3a87c09eb] database_access_failure
[2024-09-20 10:04:07] [cron:error] [cron:error] [DBIUsageException trace uid:66ed3a87c09eb] #0 /var/www/filesender/classes/data/StatLog.class.php(401): DBIStatement->__call()
[2024-09-20 10:04:07] [cron:error] [cron:error] [DBIUsageException trace uid:66ed3a87c09eb] #1 /var/www/filesender/scripts/task/cron.php(238): StatLog::clean()
[2024-09-20 10:04:07] [cron:error] [cron:error] [DBIUsageException trace uid:66ed3a87c09eb] #2 {main}
[2024-09-20 10:04:07] [cron:error] [cron:error] [DBIUsageException details uid:66ed3a87c09eb] "SQLSTATE[42601]: Syntax error: 7 ERROR:  syntax error at or near "$1"
LINE 1: ...OM StatLogs WHERE created < DATE_SUB(NOW(), INTERVAL $1 DAY)
                                                                ^"
[2024-09-20 10:04:07] [cron:error] [cron:error] [DBIUsageException details uid:66ed3a87c09eb] Array
[2024-09-20 10:04:07] [cron:error] [cron:error] [DBIUsageException details uid:66ed3a87c09eb] (
[2024-09-20 10:04:07] [cron:error] [cron:error] [DBIUsageException details uid:66ed3a87c09eb]     [method] => execute
[2024-09-20 10:04:07] [cron:error] [cron:error] [DBIUsageException details uid:66ed3a87c09eb]     [args] => Array
[2024-09-20 10:04:07] [cron:error] [cron:error] [DBIUsageException details uid:66ed3a87c09eb]         (
[2024-09-20 10:04:07] [cron:error] [cron:error] [DBIUsageException details uid:66ed3a87c09eb]             [0] => Array
[2024-09-20 10:04:07] [cron:error] [cron:error] [DBIUsageException details uid:66ed3a87c09eb]                 (
[2024-09-20 10:04:07] [cron:error] [cron:error] [DBIUsageException details uid:66ed3a87c09eb]                     [:days] => 60
[2024-09-20 10:04:07] [cron:error] [cron:error] [DBIUsageException details uid:66ed3a87c09eb]                 )
[2024-09-20 10:04:07] [cron:error] [cron:error] [DBIUsageException details uid:66ed3a87c09eb]
[2024-09-20 10:04:07] [cron:error] [cron:error] [DBIUsageException details uid:66ed3a87c09eb]         )
[2024-09-20 10:04:07] [cron:error] [cron:error] [DBIUsageException details uid:66ed3a87c09eb]
[2024-09-20 10:04:07] [cron:error] [cron:error] [DBIUsageException details uid:66ed3a87c09eb]     [query] => DELETE FROM StatLogs WHERE created < DATE_SUB(NOW(), INTERVAL :days DAY)
[2024-09-20 10:04:07] [cron:error] [cron:error] [DBIUsageException details uid:66ed3a87c09eb] )
[2024-09-20 10:04:07] [cron:error] [cron:error] [DBIUsageException details uid:66ed3a87c09eb]
```

**Comment by monkeyiq** [comment]
This should be fixed with https://github.com/filesender/filesender/pull/2014

It uses MAKE_INTERVAL which is not in older postgresql.  But I think older here means around 10 years old so we should be fine.

**Comment by monkeyiq** [comment]
This has been merged into development3 at https://github.com/filesender/filesender/blob/fcf65d7a47c1e4a25f971d6dfe1d33e3d629b319/classes/data/StatLog.class.php#L402

It should be fairly simple to cherry pick the PR into an existing codebase

**Comment by WebSpider** [comment]
I can confirm this fix works on 3.0rc3.

**Comment by monkeyiq** [comment]
Excellent
